server/src/main/java/com/talkka/server/bus/util/CenterStationConverter.java

```diff
@@ -0,0 +1,20 @@
+package com.talkka.server.bus.util;
+
+import com.talkka.server.bus.enums.CenterStation;
+
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+@Converter(autoApply = true)
+public class CenterStationConverter implements AttributeConverter<CenterStation, String> {
+
+	@Override
+	public String convertToDatabaseColumn(CenterStation attribute) {
+		return attribute.getCode();
+	}
```

Comment:
```suggestion
	@Override
	public String convertToDatabaseColumn(CenterStation attribute) {
		return attribute.getCode();
	}
```

다른 converter들은 삼항연산자로 null을 체크하던데 해당 converter에서는 null을 체크하지 않고 바로 code를 받는 이유가 뭔가요? (response를 잘 몰라서 발생한 궁금증...)

Reply:
또한 해당 메서드에 넘겨주는 매개변수의 이름을 통일시켜주면 좋을 듯 합니다!
attribute => centerStation

Reply:
Null pointer exception 나올 뻔 했네요 수정하겟슴다!
attribute도 확인해보겟습니다